website/docs/_Node/node.mdx

```diff
@@ -1,6 +1,174 @@
 ---
 id: run-node
-title: Run Node (Coming soon...)
+title: Run a Node
 sidebar_position: 7
 ---
-# Coming soon...
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+Running a CKB node boosts network robustness and decentralization while enhancing data access security.
+
+## Run a CKB Mainnet Node
+
+### Steps
```

Comment:
Before introducing the essential steps, can we explain the purpose of running  Mainnet, Testnet, and RPC nodes, and to help users identify if they are the right audience for each type of node. I feel like this may ensure that users understand the significance and requirements of each node type, making it easier for them to determine which node suits their needs best

Reply:
Agree. Below is the message to clarify this:

Choose the CKB Node that suits your purpose:
Mainnet Node: To actively participate in the live network.
Testnet Node: To experiment with blockchain features in a risk-free environment. (/// but currently we don't have guide available)
RPC Node: Set up a Nervos API endpoint for your project without running any CKB nodes yourself.

@RetricSu could you please review the above description?